contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -216,6 +216,22 @@ TsqlFunctionConvert(TypeName *typename, Node *arg, Node *style, bool try, int lo
 		 */
 		result = makeTypeCast(helperFuncCall, typename, location);
 	}
+
+	else if (strcmp(typename_string, "binary") == 0 || strcmp(typename_string, "varbinary") == 0)
+	{
+		if (style == NULL)
+			result = makeTypeCast(arg, typename, location);
+		else
+		{
+			Node	   *helperFuncCall;
+
+			args = list_make2(arg, style);
+			helperFuncCall = (Node *) makeFuncCall(TsqlSystemFuncName("babelfish_conv_helper_to_varbinary"), args, COERCE_EXPLICIT_CALL, location);
```

Comment:
`TsqlFunctionConvert` also handles behaviour of `TRY_CONVERT`, here irrespective of value of `try`, we are using `babelfish_conv_helper_to_varbinary` function which will throw error for `CONVERT` function but should return `NULL` for `TRY_CONVERT`. Kindly add appropriate handling for `TRY_CONVERT` as well.